src/swagger.json

```diff
@@ -9,13 +9,28 @@
     {
       "url": "http://localhost:3000",
       "description": "Development server"
+    },
+    {
+      "url": "https://alpha-ec-be.onrender.com/swagger",
+      "description": "production server server"
     }
   ],
+  "components": {
+    "securitySchemes": {
+      "token": {
+        "type": "apiKey",
+        "scheme":"bearer",
+        "bearerFormat": "JWT",
+        "name":"Authorization",
+        "in":"header"
+      }
+    }
+  },
   "paths": {
     "/api/users/login": {
       "post": {
         "tags": [
-          "login"
```

Comment:
Why are changing someone else's codes

Reply:
i changed this in order to give login and register same tag and we decided to do it as team